tox.ini

```diff
@@ -1,28 +1,49 @@
 [tox]
 minversion = 1.8
 envlist =
-    {py27,pypy}-{111},
-    py34-{111},
-    py35-{111},
-    py36-{111}
+    py27-{111},
+    py34-{111,20}
+    py{35,36,37}-{111,20,21,22}
 
 [testenv]
-commands = coverage run runtests.py
 deps =
-    111: Django>=1.11a1,<1.11.99
-    postgres: psycopg2
-    mysql: mysqlclient
-    coverage==4.5.3
+    mock<2.0
+    py{py,27,34}: mysqlclient<1.4
+    py{35,36,37}: mysqlclient
+    psycopg2-binary
+    111: Django>=1.11,<2.0
+    20: Django>=2.0,<2.1
+    21: Django>=2.1,<2.2
+    22: Django>=2.2,<3.0
 setenv =
+    PYTHONPATH = {toxinidir}:{toxinidir}/test_project
     DJANGO_SETTINGS_MODULE = test_project.settings
-    postgres: DJANGO_SETTINGS_MODULE = test_project.postgres_settings
     mysql: DJANGO_SETTINGS_MODULE = test_project.mysql_settings
+    postgresql: DJANGO_SETTINGS_MODULE = test_project.postgres_settings
```

Comment:
settings path are declared in commands, so DJANGO_SETTINGS_MODULE is not really useful here

Reply:
Removed